pxr/usd/usdUI/schema.usda

```diff
@@ -183,3 +188,83 @@ class Backdrop "Backdrop" (
     )
 }
 
+class "AccessibilityAPI" (
+    inherits = </APISchemaBase>
+    doc = """
+    This API describes \\em Accessibility information on a Prim that may be surfaced to a given runtimes
+    accessibility frameworks.
+    This information may be used by assistive tooling such as voice controls or screen readers.
+    Accessibility information is provided as a standard triplet of label, description and priority.
+
+    OpenUSD does not provide a accessibility runtime itself, but endeavours to provide the information needed for
+    compatible runtimes to extract and present this information.
+
+    This is a multiple apply schema, and so may have multiple namespaced accessibility triplets, where a instance
+    name may reflect a given purpose for that triplet. For example, you may desire to express different
+    information for different aspects of the prim, such as size or color.
+
+    \\note The API will use \\em default as an instance name if no namespace is specified.
+    
+    There are several best practices for using this schema.:
+
+    \\li Most accessibility runtimes support a single accessibility description. Therefore we recommend placing
+    any critical information in the default instance.
+
+    \\li A default value should be authored if using time sampled accessibility information. 
+    This helps accessibility runtimes that do not currently support time sampled information.
+
+    \\li Provide accessibility information of your scene on the default prim of the layer, and any top level prims.
+    This allows accessibility systems to provide concise scene descriptions to a user, but also allows supporting
+    accessibility systems that either do not support hierarchy information or when a user has turned off that level
+    of granularity. Accessibility information may still be provided on other prims in the hierarchy.
+
+    \\note The use of the default prim and top level prims for scene accessibility descriptions is a
+    recommended convention. Outside of that, accessibility information is not implicitly inherited through
+    a prim hierarchy. The inheritance should be left to the accessibility runtime to decide how best to
+    surface information to users.
+    """
+
+    customData = {
+        token apiSchemaType = "multipleApply"
+        token propertyNamespacePrefix = "accessibility"
+        dictionary schemaTokens = {
+            dictionary label = {
+                string value = "label"
+                string doc = """Name of the label attribute"""
+            }
+            dictionary description = {
+                string value = "description"
+                string doc = """Name of the description attribute"""
+            }
+            dictionary priority = {
+                string value = "priority"
+                string doc = """Name of the priority attribute"""
+            }
+        }
+    }
+) {
+    string label (
+        doc = """A short label to concisely describe the prim.
```

Comment:
Any guidance on "shortness" or other constraints (like avoiding new lines) that you'd recommend for label?

Asking from the perspective of validation.

Reply:
Unfortunately I don't believe there's any standard length.
It's just meant to be shorter than the description.